### PR TITLE
Allows the renderer tags to be used without a many-many relation

### PR DESCRIPTION
**Pull request by Seb-C**

### CODE REVIEW

**Comment by jguyomard** [comment]
:+1:

**Review by jguyomard — APPROVED** [review]

**Comment by tonyyb** [comment]
:+1:

File: framework/classes/renderer/tag.php [hunk 1]
```diff
@@ -54,7 +54,10 @@ public function before_save($item, $data)
 
         $tag_class = $this->renderer_options['model'];
 
-        $item->{$this->renderer_options['relation_name']} = $tag_class::find('all', array('where' => array(array($this->renderer_options['label_column'], 'IN', array_keys($tags)))));
+        $relation = $item->relations($this->renderer_options['relation_name']);
+        if ($relation instanceof \Orm\ManyMany) {
+            $item->{$this->renderer_options['relation_name']} = $tag_class::find('all', array('where' => array(array($this->renderer_options['label_column'], 'IN', array_keys($tags)))));
```
Comment (shaoshiva) [inline framework/classes/renderer/tag.php:59]: Code style

```
item->{$this->renderer_options['relation_name']} = $tag_class::find('all', array(
    'where' => array(
        array($this->renderer_options['label_column'], 'IN', array_keys($tags))
    )
));
```

**Comment by shaoshiva** [comment]
Maybe the relation should be emptied for non ManyMany relations ?

Otherwise :+1:

**Pull request merged**